packages/connect-react/src/components/ConfigureFilePicker/index.tsx

```
@@ -783,80 +1007,219 @@ export const ConfigureFilePicker: FC<ConfigureFilePickerProps> = ({
+                  {items.map((item) => {
+                    const selected = isSelected(item);
+                    // Show checkbox at file/folder level for selectable items
+                    const canSelect = currentProp === fileOrFolderProp && (
+                      (item.isFolder && selectFolders) || (!item.isFolder && selectFiles)
+                    );
+
+                    return (
+                      <li
+                        key={item.id}
+                        role="button"
+                        tabIndex={0}
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)